.github/workflows-source/schedule-daily.yml

```diff
@@ -230,3 +230,82 @@ jobs:
           compression-level: 9
           path: |
             cov_html.zip
+
+  recovery-engine-smoke-test:
+    name: Recovery Engine Smoke Test
+    <<: *dind-large-setup
+    timeout-minutes: 60
+    steps:
+      - <<: *checkout
+
+      - name: Write AWS credentials
```

Comment:
would prefer to avoid using credentials in tests; we are actually planning on not making them available at all soon!